modules/ROOT/pages/creating-example-archetypes.adoc

```diff
@@ -60,12 +58,12 @@ mvn org.mule.tools:mule-example-archetype:3.5.0:create ...
 +
 [NOTE]
 ====
-*artifactId* +
+artifactId
 
-As of Mule 2.2 the artifactId can contain characters such as underscore or hyphen. However, the plug-in will convert the name into a usable form suitable for Java. For example, if the argument is specified as **-DartifactId=My#Awesome-Mule_Project**, the project is created in a directory of that name, but the project name will be *MyAwesomeMuleProject* and the package name is *.myawesomemuleproject*
+As of Mule 2.2 the artifactId can contain characters such as underscore or hyphen. However, the plug-in will convert the name into a usable form suitable for Java. For example, if the argument is specified as `-DartifactId=My#Awesome-Mule_Project`, the project is created in a directory of that name, but the project name will be `MyAwesomeMuleProject` and the package name is `.myawesomemuleproject`.
```

Comment:
Here plug-in is hyphenated but above it wasn't. SF Style Guide says it should be plug-in, so you may want to S&R. 
I think we need tick marks for `artifactId`